server/controllers/signin.js

```diff
@@ -0,0 +1,33 @@
+import { Pool } from 'pg';
+import token from '../helpers/jwt';
+import DbQuery from '../model';
+
+import { hashPwd } from '../helpers/pwdHash';
+
+export default class signInUser {
+  static async signin(req, res) {
```

Comment:
@olamilekan000 Can you explain to me what is going on here?

Reply:
The hasPwd that was imported should be there and also I am exporting the signInUser class